src/plugins/autoflow/AutoflowTabularPluginSpec.js

```diff
@@ -264,15 +264,17 @@ xdescribe("AutoflowTabularPlugin", () => {
                 });
             });
 
-            it("displays historical telemetry", () => {
+            fit("displays historical telemetry", () => {
```

Comment:
@davetsay i was working on this suite

Reply:
@unlikelyzero , fixed the autotabular specs